gtsam/geometry/Similarity3.cpp

```diff
@@ -24,7 +24,7 @@
 namespace gtsam {
 
 using std::vector;
-using PointPairs = vector<Point3Pair>;
+typedef vector<Point3Pair> PointPairs;
```

Comment:
Why not Point3Pairs? I understand its not related to your change, but I dont know why we dropped the 3 there.

Reply:
I agree that would be better. Yeah just legacy code, we could improve it likely

Reply:
Move this typedef to the header

Reply:
thanks @ProfFan , just moved it to the `Similarity3.h` file

Reply:
should be updated now to `Point3Pairs`